__tests__/data/satismeter_source.json

```diff
@@ -0,0 +1,455 @@
+                "os": "macOS",
+                "type": "desktop"
+              },
+              "page": {
+                "referrer": "http://127.0.0.1:5500/testSm.html"
+              }
+            },
+            "integrations": {
+              "SATISMETER": false
+            },
+            "type": "track",
+            "traits": {
+                "groups": {"group1": "grooupId"}
+            },
+            "userId": "No response",
+            "properties":{
+                "category": "detractor",
+                "answers": [
+                    {
+                      "label": "How likely are you to recommend us to your friends and colleagues?",
+                      "id": "7ddb22b0-64a8-11ed-a4c7-b3bed73771cd",
+                      "value": 5,
+                      "name": "SM_rating",
+                      "type": "scale",
+                      "metric": "nps"
+                    },
+                    {
+                      "label": "What could we do to improve?",
+                      "id": "7ddb22b1-64a8-11ed-a4c7-b3bed73771cd",
+                      "value": "Many things to imporve\n",
+                      "name": "SM_comment",
+                      "type": "long-text"
+                    },
+                    {
+                      "label": "The company made it easy for me to handle my issue.",
+                      "id": "1dc53f60-66a0-11ed-856c-6f39711bf041",
+                      "value": 4,
+                      "name": null,
+                      "type": "scale",
+                      "metric": "ces"
+                    },
+                    {
+                      "label": "How satisfied were you with the service you received?",
+                      "id": "24c5b290-66a0-11ed-856c-6f39711bf041",
+                      "value": 4,
+                      "name": null,
+                      "type": "smiley",
+                      "metric": "csat"
+                    },
+                    {
+                      "label": "How you like to rate the surevy?",
+                      "id": "27b3d1d0-66a0-11ed-856c-6f39711bf041",
+                      "value": 4,
+                      "type": "scale"
+                    },
+                    {
+                      "label": "Your Name (Single Answer)",
+                      "id": "37a8c000-66a0-11ed-856c-6f39711bf041",
+                      "value": "a",
+                      "type": "single-choice"
+                    },
+                    {
+                      "label": "Your Name (Multiple Answer)",
+                      "id": "4b435da0-66a0-11ed-856c-6f39711bf041",
+                      "value": [
+                        "a1",
+                        "b1"
+                      ],
+                      "type": "multiple-choice"
+                    }
+                  ],
+                "surveyDismissed":false,
+                "surveyPending": false,
+                "receivedAt":"2022-11-17T17:51:21.764Z"
```

Comment:
Can't we put `receivedAt` to root?

Reply:
`receivedAt` at the root level mainly is regarding when the event was received by the rudder-server and what we want to convey is when the response was recorded by Satismeter.
Ref: https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/#:~:text=client%20to%20RudderStack.-,receivedAt,-The%20timestamp%20when